cmd/minikube/cmd/config/config.go

```diff
@@ -105,9 +105,14 @@ var settings []Setting = []Setting{
 
 var ConfigCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Modify minishift config",
-	Long: `config modifies minishift config files using subcommands like "minishift config set vm-driver kvm"
-Configurable fields: ` + "\n\n" + configurableFields(),
+	Short: "Modifies Minishift configuration properties.",
+	//NEEDINFO: can we do this any time? or do we need to stop the cluster first? how many config files can we have?
+	Long: `Modifies Minishift configuration properties. The configuration properties in the file are parallel to
+	the options that you set when you run the `minishift start` command. You can set the configuration in
+	this file and point to it during cluster start instead of setting the options directly in the command.
```

Comment:
> You can set the configuration in this file and point to it during cluster start instead of setting the options directly in the command

Where did you get this from?